src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -8,16 +8,17 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import mate.jdbc.dao.ManufacturerService;
 import mate.jdbc.exception.DataProcessingException;
 import mate.jdbc.lib.Dao;
 import mate.jdbc.model.Manufacturer;
 import mate.jdbc.util.ConnectionUtil;
 
 @Dao
-public class ManufacturerDaoImpl implements ManufacturerDao {
+public class ManufacturerDaoImpl implements ManufacturerService {
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        String query = "INSERT INTO manufacturers (name, country) "
+        String query = "INSERT INTO manufacturer (name, country) "
```

Comment:
Get back all tese 's'

Reply:
names of database tables that are used in this and previous tasks without 's'